samples/emoji-search/presenters/src/jvmMain/kotlin/app/cash/zipline/samples/emojisearch/RealHostApi.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.samples.emojisearch
+
+import java.io.IOException
+import kotlin.coroutines.suspendCoroutine
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.Headers.Companion.toHeaders
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+
+class RealHostApi : HostApi {
+  private val client = OkHttpClient()
+  override suspend fun httpCall(url: String, headers: Map<String, String>): String {
+    return suspendCoroutine { continuation ->
```

Comment:
Should really be `suspendCancelableCoroutine` for correctness.

Reply:
Fixed.